web/src/styles/global.css

```diff
@@ -52,6 +51,18 @@
         top: 130%;
         left: 0;
     }
+    .text-card-accent {
+        position: relative;
+    }
+    .text-card-accent:after {
+        content: '';
+        height: 5px;
+        width: 50px;
+        background-color: #acc9e8;
+        position: absolute;
+        top: 20%;
+        left: 0%;
+    }
```

Comment:
If we're not using this let's remove it.

Reply:
@asasmith removed on a different PR

Reply:
@kcbreuil can you link that pr here? It looks like this code was added on this branch so I don't know how/why it would be on any other branches.

Reply:
@kcbreuil this looks like new code on this pr though. Do you see it added in a different pr?